src/stores/PreferenceStore.ts

```diff
@@ -144,7 +144,7 @@ export class PreferenceStore {
 
     private getASTGridVisible = (): boolean => {
         const astGridVisible = localStorage.getItem(PREFERENCE_KEYS.astGridVisible);
-        return astGridVisible === "false" ? false : DEFAULTS.astGridVisible;
```

Comment:
not sure why this line needs to change

Reply:
because now default is changed from true to false, so the statement needs to be flipped accordingly, thought about modified to only `return astGridVisible === "true";`, which is better @veggiesaurus  u think? XD

Reply:
if that's the case, then you're not really relying on DEFAULTS.astGridVisible properly... which is not a sensible way to do it. Rather:
`astrGridVisible === "true"? true: (astGridVisible === "false"? false: DEFAULTS.astGridVisible)`. This way, you only resort to the default value if the value is neither "true" nor "false"

Reply:
you might want to make a `parseBoolean(value: string, defaultValue: bool)` utility function for this.

Reply:
oh yeah that's a good idea to decouple!  will add it